src/store/modules/picklist/actions.ts

```diff
@@ -53,7 +53,8 @@ const actions: ActionTree<PicklistState, RootState> = {
     let resp;
     const params = {
       "inputFields": {
-        "statusId": "PICKLIST_COMPLETED",
+        "statusId": ["PICKLIST_COMPLETED", "PICKLIST_PICKED"],
```

Comment:
We need to add the check for PICKLIST_PICKED on other places as well where we have checks for completed status.